File: src/main/java/net/yakclient/mixin/internal/methodadapter/core/CoreABPatternMatcher.java
package net.yakclient.mixin.internal.methodadapter.core;

import net.yakclient.mixin.internal.instruction.Instruction;
import org.objectweb.asm.MethodVisitor;

public class CoreABPatternMatcher extends CoreMixinPatternMatcher {
    public CoreABPatternMatcher(MethodVisitor visitor, Instruction instructions) {
        super(visitor, instructions);
    }

    @Override
    public void visitInsn() {
    }

    @Override
    public void visitCode() {
        super.visitCode();

        this.executeInsn();
    }
}

File: base/src/main/kotlin/net/yakclient/mixins/base/internal/methodadapter/core/CoreABPatternMatcher.kt
package net.yakclient.mixins.base.internal.methodadapter.core

import net.yakclient.mixins.base.internal.instruction.Instruction
import org.objectweb.asm.MethodVisitor

class CoreABPatternMatcher(
    visitor: MethodVisitor,
    instructions: Instruction
) : CoreMixinPatternMatcher(visitor, instructions) {

    override fun visitInsn() {}

    override fun visitCode() {
        super.visitCode()
        this.executeInsn()
    }
}